Dockerfile

```diff
@@ -1,15 +1,15 @@
 # Base layer
-FROM ubuntu:20.04 as baseline
+FROM ubuntu:22.04 as baseline
 RUN apt update && apt upgrade -y \
   && apt install -y python3 python3-dev python3-pip curl unzip gnupg \
   && update-alternatives --install /usr/bin/python python /usr/bin/python3 1 \
   && update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
 
 # Layers used for building/downloading/installing tools
 FROM baseline as tool_builder
-ARG HELM_VERSION=3.8.1
+ARG HELM_VERSION=3.10.0
 ARG KUBECTL_VERSION=1.23.8
-ARG TERRAFORM_VERSION=1.2.0
+ARG TERRAFORM_VERSION=1.3.2
```

Comment:
Does oss-k8s require features present in TF 1.3.2 or is that the version you've been using locally? The other providers still reference TF 1.0.0 in their Dockerfiles.

Reply:
Just noticed that the Readme calls out TF 1.3.2 as required below.